support/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -105,6 +105,92 @@ Note: If you would like this tracking domain to be the default, please add `"def
+
+    * Under Distribution Settings, fill in **Alternate Domain Names (CNAMEs)** with your custom tracking domains (i.e., www.customtrackingdomain.com).	
+        * **Note:** Can add up to 100 domains.	
+
+        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_cnames.png)	
+
+    * Under Default Cache Behavior Settings, set **Query String Forwarding and Caching** to "Forward all, cache based on all".	
+
+        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_query_string_forward.png)	
+
+    * Create, or update, a CNAME record with your DNS service to route queries for tracking domain(s) with your CloudFront distribution ID.	
+
+    * Under Distribution Settings, for "SSL Certificate", select **Custom SSL Certificate** - Upload certificates as needed.	
+
+        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_custom_ssl_cert.png)	
+
+    * At the bottom of the page, press **Create Distribution**.	
+
+        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_create_distribution.png)	
+
+5. On the **Create Distribution** page, under Default Cache Behavior Settings, it is recommended to set a custom whitelist for the `User-Agent` header.  Set the configuration **Cache Based on Selected Request Headers** to "Whitelist".  At this point you will need to "Add Custom" for `User-Agent`.  This allows `User-Agent` data to be present in your engagement events received from SparkPost.	
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_whitelist_cache.png)	
+
+6. Log into SparkPost app use the SparkPost [Tracking Domain API](https://developers.sparkpost.com/api/tracking-domains/#tracking-domains).	
+
+7. Add **custom tracking domain** and verify the domain.	
+
+8. Use tracking domain while sending messages.	
+
+### Updating an Existing Domain on AWS CloudFront	
+
+If you use AWS CloudFront to enable SSL engagement tracking, by default, CloudFront replaces the User-Agent header with "Amazon CloudFront," obscuring device and client information. To regain the device and client information, configure CloudFront to forward the original User-Agent header.	
+
+1. Navigate to the CloudFront console.	
+
+2. Click on your distribution's ID.	
+
+      ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_edit_domain_distribution_id.png)	
+
+3. Click on the "Behaviors" tab.	
+
+      ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_edit_domain_behaviors_tab.png)	
+
+4. Click on the checkmark next to the first item in the list.	
+
+      ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_edit_domain_checkmark.png)	
+
+5. Click on the "Edit" button.	
+
+      ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_edit_domain_edit_button.png)	
+
+6. In the dropdown next to "Cache Based on Selected Request Headers," select "All."	
```

Comment:
I think this section needs to be updated with the "Cache based on Selected Requested Headers" like it is above.  We should copy the paragraph above as this will break with the new ET service.

'It is recommended to set a custom whitelist for the `User-Agent` header.  Set the configuration **Cache Based on Selected Request Headers** to "Whitelist".  At this point you will need to "Add Custom" for `User-Agent`.  This allows `User-Agent` data to be present in your engagement events received from SparkPost.'